sdk/aqueduct/api_client.py

```diff
@@ -311,6 +313,14 @@ def get_workflow(self, flow_id: str) -> GetWorkflowResponse:
         workflow_response = GetWorkflowResponse(**resp.json())
         return workflow_response
 
+    def get_workflow_writes(self, flow_id: str) -> GetWorkflowWrittenObjectsResponse:
```

Comment:
referring to our naming guide https://www.notion.so/aqueducthq/SDK-External-API-Naming-ce685c0980ae4d7ebee569bf53ef2a01 this should method should be named as `list_saved_objects`, and agree that it should belongs to `flow` object